solid_utils/sampling.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import math
+import warnings
+from .variogram import remove_trend as rt
+def load_geo(attr_geo):
```

Comment:
add a few lines to separate between module imports and functions please.

Reply:
Sure.